.github/workflows/build.yml

```diff
@@ -28,10 +28,15 @@ jobs:
               bindleUrl: "https://bindle.blob.core.windows.net/releases/bindle-v0.8.0-linux-amd64.tar.gz",
               bindleBinary: "bindle-server",
               pathInBindleArchive: "bindle-server",
+              nomadUrl: "https://releases.hashicorp.com/nomad/1.3.1/nomad_1.3.1_linux_amd64.zip",
+              nomadBinary: "nomad",
+              pathInNomadArchive: "nomad",
+              hippoUrl: "https://github.com/deislabs/hippo/releases/download/v0.17.0/hippo-server-linux-x64.tar.gz",
               wasmtimeUrl: "https://github.com/bytecodealliance/wasmtime/releases/download/v0.36.0/wasmtime-v0.36.0-x86_64-linux.tar.xz",
               wasmtimeBinary: "wasmtime",
               pathInWasmtimeArchive: "wasmtime-v0.36.0-x86_64-linux/wasmtime",
               platformAgnosticChecks: true,
+              isNotWindows: true
```

Comment:
Can we use the pre-existing `isWindows` key instead?  eg `isWindows: false` here and below?

Reply:
I update the if condition in steps, so we don't need to introduce both `isWindows` and `isNotWindows`.